Description: Fix for libpng 1.6 new API
Author: Tobias Frost <tobi@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=809837
Last-Update: 2016-01-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sources/png.c
+++ b/sources/png.c
@@ -5,6 +5,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include "to_png.h"
+#include <zlib.h>
 #include <png.h>
 
 #define PDNCOL 256
@@ -36,7 +37,7 @@
 	}
 
 	/* set up error handlimg for libpng */
-	if (setjmp(png_ptr->jmpbuf)) {
+	if (setjmp(png_jmpbuf(png_ptr))) {
 		png_destroy_write_struct(&png_ptr, &info_ptr);
 		return;
 	}
